src/Api/SecretManagerFeatures/Models/Response/ProjectAccessPoliciesResponseModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Models.Api;
+
+namespace Bit.Api.SecretManagerFeatures.Models.Response;
+
+public class ProjectAccessPoliciesResponseModel : ResponseModel
+{
+    public ProjectAccessPoliciesResponseModel(IEnumerable<BaseAccessPolicy>? baseAccessPolicies,
+        string obj = "projectAccessPolicies") : base(obj)
+    {
+        if (baseAccessPolicies == null) return;
```

Comment:
Please use the full syntax.
```suggestion
        if (baseAccessPolicies == null) {
          return;
        }
```

Reply:
Resolved in https://github.com/bitwarden/server/pull/2507/commits/1615d246d7a67fdaeae21939e786d99d43e04c3d